src/main/telemetry/crsf.c

```diff
@@ -266,19 +266,35 @@ void crsfFrameFlightMode(sbuf_t *dst)
     uint8_t *lengthPtr = sbufPtr(dst);
     sbufWriteU8(dst, 0);
     sbufWriteU8(dst, CRSF_FRAMETYPE_FLIGHT_MODE);
-
-    // use same logic as OSD, so telemetry displays same flight text as OSD
-    const char *flightMode = "ACRO";
-    if (airmodeIsEnabled()) {
-        flightMode = "AIR";
-    }
-    if (FLIGHT_MODE(FAILSAFE_MODE)) {
-        flightMode = "!FS";
-    } else if (FLIGHT_MODE(ANGLE_MODE)) {
-        flightMode = "STAB";
-    } else if (FLIGHT_MODE(HORIZON_MODE)) {
-        flightMode = "HOR";
+    const char *flightMode = "";
+    
+    if (ARMING_FLAG(ARMED)) {
```

Comment:
This breaks the functionality for existing users that expect the modes to be shown when not armed. Also, there is no point to send a meaningless response instead of the actual mode that is selected. Please change this back.

Reply:
@mikeller I'm missing what you mean by the meaningless response.  Do you mean the `OK` status?  That's the most critical part of this PR.

Reply:
Yes. This does not convey any useful information.

Reply:
@mikeller Being armed or not armed isn't important?  I don't mean to debate this, if knowing armed state is not useful to the Betaflight community I can totally accept that.  I'm not trying to send useless information, it's actually the main thing I'm trying to do in this PR, note armed/unarmed status.  Without armed state, to me at least, this PR is useless.  But like I said, I have no interest in debating the merits of this, just giving my perspective.

Reply:
- you just said 'ready to arm' was the important bit. now it's 'armed'. Which one is it?
- I would hope that pilots flying Betaflight are always aware if being armed or not, otherwise they should not be flying;
- 'armed' is obviously a status that is orthogonal to the flight modes, so it should be tracked separately, and not used to override safety relevant flight mode information.

In stating my last point. since the flight mode is encoded in a text field, it should be easy to encode the arming status by appending a character like '!' to every status when armed. This would allow for a much more versatile script without losing important information or compromising on human readability.

Reply:
@mikeller Sorry, what I mean is that the most critical is knowing armed status, which includes ready to arm, armed, error, waiting, etc.  This is the trigger to know that the flight has started.  Without armed status, timers and other statuses can't be triggered.

The issue here is that Crossfire doesn't provide an armed status, so this PR was designed to enhance the existing flight mode information with additional information, more flight modes, error, waiting, ready to arm, etc.

Adding a special character was considered initially when added to INAV, as well as adding many more messages (like the actual reason for an error for example).  It was decided to make it more human readable by just showing the flight modes instead of a bunch of codes and symbols.  It was decided to make them match the OSD as much as possible.

Again, I'm not trying to debate this, just giving my perspective.  If this PR is unacceptable, I totally understand.  Not the first time a PR of mine was rejected, no offence taken.